__tests__/app.test.js

```diff
@@ -0,0 +1,48 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data/index");
+
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+  test("200: response array should contain all of the data from topics.", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(topics.length).toBe(3);
+        topics.forEach(({ description, slug }) => {
```

Comment:
You can also change this to a .toMatchObject which is a really useful jest matcher. Also, remember that description is not set at NOT NULL when the tables were created so may not always be a string.